concrete/attributes/topics/controller.php

```diff
@@ -347,14 +366,27 @@ public function getTopicTreeID()
         return $this->akTopicTreeID;
     }
 
+    public function getAllowMultipleValues()
```

Comment:
We should come up with a standard for how we name getters that are booleans, I probably would've done `allowsMultipleValues`